src/css.js

```diff
@@ -257,7 +257,9 @@ jQuery.extend( {
 			if ( !hooks || !( "set" in hooks ) ||
 				( value = hooks.set( elem, value, extra ) ) !== undefined ) {
 
-				style[ name ] = value;
+				if ( support.styleWhitespace || !/^\s+$/.test( value ) ) {
+					style[ name ] = value;
+				}
 			}
```

Comment:
> Is there harm in just punting on whitespace only string across all enviros without a need to add a support test?

I was thinking about it, hence the: "Another option would be to skip the support test and just apply the workaround everywhere" at the end of the PR description. :) I just wanted to try the support-test way first to not change anything outside of IE 11 but I can be persuaded otherwise.

> You can simplify the check to something like `if (value && value == 0) {`. It takes advantage of the fact that
> `" " == 0` is `true`.

The regex is more descriptive and using `==` is against our ESLint config so I'd have to add an exception. I'd rather not do that when not necessary.

So it could be:
```js
if ( value && /^\s+$/.test( value ) ) {
	style[ name ] = value;
}
```